contracts/multisig/src/contract.rs

```diff
@@ -46,7 +46,7 @@ pub fn execute(
         ExecuteMsg::KeyGen {
             key_id,
             snapshot,
-            pub_keys,
+            pub_keys_by_address: pub_keys,
```

Comment:
Why this renaming? If the renaming is legit, it should remain this way entering the execute mod.